cpp/log/log_signer.h

```diff
@@ -53,7 +53,8 @@ class LogSigner : public cert_trans::Signer {
   SignResult SignTreeHead(ct::SignedTreeHead* sth) const;
 
  private:
-  static SignResult GetSerializeError(SerializeResult result);
+  static SignResult GetSerializeError(
+      cert_trans::serialization::SerializeResult result);
```

Comment:
Ugh, not your fault, but if only this (and a few other) header(s) had all been in the `cert_trans` namespace, it'd have been a bit smoother... :disappointed: